src/box/lua/config/init.lua

```diff
@@ -601,6 +601,10 @@ function methods.instance_uri(self, uri_type, opts)
     return self._configdata_applied:_instance_uri(uri_type, uri_opts)
 end
 
+function methods.jsonschema(_self)
+    return cluster_config:jsonschema()
+end
```

Comment:
Other methods call the `selfcheck` utility function. I would do it here as well.

Reply:
Added.